recovery/src/androidTest/java/com/kin/ecosystem/recovery/qr/QRBarcodeGeneratorImplTest.java

```diff
@@ -73,10 +79,16 @@ public void generate_success() throws Exception {
 
 	@Test
 	public void decodeQR_success() throws Exception {
-		Bitmap bitmap = TestUtils.loadBitmapFromResource(this.getClass(), "qr_test.png");
+		// test a QR that was being able to be decoded only with TRY_HARDER flag on in zxing library
+		Bitmap bitmap = TestUtils.loadBitmapFromResource(this.getClass(), "backup_qr_try_harder.png");
```

Comment:
backup_qr_try_harder.png is the QR that didn't succeed before?

Reply:
yep